client/src/app/site/common/components/legal-notice/legal-notice.component.ts

```diff
@@ -1,12 +1,50 @@
 import { Component, OnInit } from '@angular/core';
 
+import { Observable } from 'rxjs';
+
+import { CountUsersStatisticsService, CountUserStatistics } from '../../services/count-user-statistics.service';
+import { auditTime } from 'rxjs/operators';
+
 @Component({
     selector: 'os-legal-notice',
-    templateUrl: './legal-notice.component.html',
-    styleUrls: ['./legal-notice.component.scss']
+    templateUrl: './legal-notice.component.html'
 })
 export class LegalNoticeComponent implements OnInit {
-    public constructor() {}
+    public token: string = null;
+    public stats: CountUserStatistics = null;
+
+    public constructor(private countUsersStatisticService: CountUsersStatisticsService) {}
 
     public ngOnInit(): void {}
+
+    public countUsers(): void {
```

Comment:
should get an own page if you want to leave that code

Reply:
ok